### DISCUSSION TITLE
Win11 difficulties using Mountainduck Taskbar Icon to Connect, Disconnect or anything else

### DISCUSSION THREAD

**Discussion by Antergosgeek** [iterate-ch/cyberduck#16629] [Q&A]
On Windows 11, when I click on the Mountainduck taskbar icon to connect to one of my bookmarks, when I navigate to a folder then the bookmark itself, it is all ok as long as it opens the next window to the right. Anytime a window opens to the left, you cannot actually navigate to it. 

For example if it looks like this:
<img width="364" alt="image" src="https://github.com/user-attachments/assets/c94f4d30-69f1-47da-aa2f-1ad3c489ecea">

You are unable to get to and click on any of the menu items (Dropbox, Google Drive, OneDrive).
So currently the only way to connect for me is to go to my Cyberduck Bookmark folder  ```%AppData%\Cyberduck\Bookmarks``` and then right click on a bookmark and "Open with" and then find my Mountainduck executable. 

Anyone else having the same problem? Any way to solve it?

**Comment by dkocher** [comment]
Seems to be caused by a vertical taskbar no longer officially supported in Windows 11.

**Reply by Antergosgeek** [reply]
Unfortunately not. Taskbar at the bottom and horizontal...

**Comment by AliveDevil** [comment]
In case this is in the Tray Flyout (Arrow Up) to the left of any tray icon, we once in a blue moon receive your issue description, but have neither successfully replicated nor found a cause for it - it may be some weird interaction in Windows or a third-party application.

One thing that works is hitting the Tab-key once when hovering over a bookmark, then you can move the mouse over, or continue with keyboard-navigation (tab/arrow keys).

**Reply by Antergosgeek** [reply]
Thank you, will try that!

**Comment by gtflip** [comment]
I see this on Windows 10 as well.  I have to use the arrow keys to get to the menu items.  Sometimes it leaves room to the right, and I can click on that menu.  However, I have labels on all my bookmarks, so it just opens the list of bookmarks for that label, and hovering over a bookmark brings up a menu to the left of the current one, which I can't reach.

It looks to me like when you move the mouse to the left of a menu window, it removes focus from the current item, which immediately dismisses the menu it just popped up.

For reproducing it, you could try using a long label name.  Maybe that would be enough to force the issue.

**Reply by AliveDevil** [reply]
Seems to be this way, but I’ve never been able to replicate it - neither when the menu items open left or right of the current menu popup.
The only time I successfully replicated the closing menu was when I moved the pointer so slow, that a hit test fell between the new menu item and the underlying menu, effectively closing the new submenu.

**Comment by RhoGit** [comment]
I can confirm the same issue.

When the tray submenu opens to the **left side**, it is almost impossible to interact with it using the mouse. As soon as I move the cursor toward the submenu, the parent menu loses focus and the submenu closes. When the submenu opens to the **right side**, everything works normally.

This makes the tray menu difficult to use because several connections are only reachable through nested submenus.

System information:
Windows 11 Pro 24H2
OS Build: 26100.7840
Windows Feature Experience Pack: 1000.26100.291.0

It would be great if this could be investigated, as it appears to be reproducible.
